bolt.yaml

```diff
@@ -0,0 +1,20 @@
+# puppetdb:
+#   server_urls: ["https://foo.example.com:8081"]
+#   cacert: /path/to/ca.pem
+#   token: ~/.puppetlabs/token
+# pcp:
+#   service-url: "https://foo.example.com:8143"
+#   cacert: /path/to/ca.pem
+#   task-environment: production
+# ssh:
```

Comment:
For most skeleton files I added some commented out examples to stub out an idea for making a working example project that actually uses all the example files. Is that worth investing the time in to? Or do we want to limit this to simply a directory structure?

Reply:
I think that if we're going to have example content in these files, we should include some comments at least 
 explaining what the sections are for.

Reply:
If the main purpose for this is to have something folks can clone and get started with at contributor summit I don't think content is necessary. People will be familiar with the file names and can refer to docs, and we'll have lots of Bolt knowledgeable folks on hand to help. I think just having empty files is fine, and we can add content if we want to reuse this for something else.